feat: cleanup obsolete address and contact flags (LAN-780)

scdanieli opened the pull request:
Only various LANDA member reports are affected by this. If there are multiple addresses for a LANDA member, https://github.com/alyf-de/landa/blob/3f2e14161390248e0c18a5eebc1ddcaf52eb5d82/landa/organization_management/report/member/member.py#L43-L49 is used to determine which one is displayed.

There is currently no possibility to explicitly display an address preferentially.

scdanieli committed:
feat: cleanup obsolet address and contact flags (LAN-780)

scdanieli committed:
chore: make semgrep happy

"label":\s*"([^"]*)" becomes "label": _("$1")

scdanieli committed:
chore: make semgrep happy again

barredterra committed:
refactor: avoid unnecessary casting

barredterra committed:
feat: don't show disabled address in report

barredterra commented on landa/water_body_management/report/members_in_water_body_management/members_in_water_body_management.py:
todo @barredterra : review error in this report

barredterra committed:
fix: drop rows with empty index

Report "Members in Water Body Management":
Drop rows with empty index (member) column. Otherwise they could not be
concatenated with the other dataframes later.

barredterra committed:
refactor: remove redundant code

No point in sorting columns here, the df is reindexed later anyway.

barredterra committed:
fix: "Awards" column label

Report "Members with Member Functions"

barredterra commented:
:tada: This PR is included in version 0.11.0 :tada:

The release is available on [GitHub release](https://github.com/alyf-de/landa/releases/tag/v0.11.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
